runner/pipeline/pipeline_cache.py

```diff
@@ -16,7 +16,7 @@ def get_pipeline(pipeline):
             resolved_dict = _pipeline.get("app")
         else:
             resolver_class = PipelineCache._get_pipeline_resolver(pipeline.pipeline_type)
-            resolver = resolver_class(pipeline.github, pipeline.entrypoint, pipeline.version)
+            resolver = resolver_class(pipeline.github, pipeline.entrypoint, pipeline.nfcore_template, pipeline.version)
```

Comment:
Can you re-order params so nfcore_template is last one and put nfcore_template as param with default value, since I am not sure this will work for CWL pipelines.